src/pager/events.rs

```diff
@@ -26,6 +33,21 @@ impl Pager {
                     KeyCode::Home => self.home(),
                     KeyCode::End => self.end(reader)?,
                     KeyCode::Esc | KeyCode::Char('q') => self.exit(),
+                    KeyCode::Enter => match self.command_line.mode {
+                        Mode::Search => self.view.search = self.command_line.input.clone(),
+                        Mode::Goto => {
+                            let input = self.command_line.input.clone();
+                            self.command_line.input.clear();
+                            let mut iter = input.split(":");
+                            iter.next()
+                                .and_then(|s| s.parse::<usize>().ok())
+                                .and_then(|y| Some(self.view.scroll_row = y.saturating_sub(1)));
+                            iter.next()
+                                .and_then(|s| s.parse::<usize>().ok())
+                                .and_then(|x| Some(self.view.scroll_col = x.saturating_sub(1)));
```

Comment:
This can probably be abstracted away into a helper function. The same logic is used while parsing the filename argument

Reply:
Fixed in 53e7fe5